query.go

```diff
@@ -158,6 +158,15 @@ func syncInfo(pkgS []string, flags []string) (err error) {
 		return
 	}
 
+	//repo always goes first
+	if len(repoS) != 0 {
+		err = passToPacman("-Si", repoS, flags)
+
+		if err != nil {
+			return
```

Comment:
If -Si errors don't you think we, should continue on anyway?

Reply:
I don't think we should, I don't see any circumstance in which we should continue in case of error
Could you point it out to me? Or aren't they just the same?

Reply:
I was thinking how pacman has an exit status of 1 if any of the packages fed into -Si don't exist. But i forgot that's handled earlier on with  `aurS, repoS, err := packageSlices(pkgS)`. So yeah I think you're right, we should exit on error.